packages/visualizations/src/components/Map/MapRender.svelte

```diff
@@ -32,6 +33,18 @@ TODO:
     let legendVariant;
     $: legendVariant = clientWidth <= 375 ? 'fluid' : 'fixed';
 
+    // Used to render tooltips on hover
+    export let renderTooltip;
+    const hoverPopup = new maplibregl.Popup({
+        closeOnClick: false,
+        closeButton: false,
+        className: 'tooltip-on-hover',
+    }).trackPointer();
+    // Used to store fixed tooltips displayed on render
+    const fixedPopupsList = [];
+    // Used to select shapes to activate a tooltip on render
+    export let activeShapes;
+
     // aspect ratio based on width, by default equal to 1
     export let aspectRatio = 1;
     $: cssVarStyles = `--aspect-ratio:${aspectRatio};`;
```

Comment:
You can use `<Component --aspect-ratior={aspectRatio} />` https://svelte.dev/docs#template-syntax-component-directives---style-props

Reply:
(maybe not such a great idea since it add a `<div>`after all)